src/main/java/dev/jbang/cli/Alias.java

```diff
@@ -47,31 +94,50 @@ static void printAliases(PrintWriter out, String catalogName, AliasUtil.Aliases
 						.stream()
 						.sorted()
```

Comment:
should we add a "<path catalog>" somewhere in this print out (at least when --verbose) to let users grok which catalogs are being consulted.

Reply:
I was actually planning on adding a `--show-origin` or `--show-location` flag to the `alias list` command (it's what `git config` does as well). But unfortunately the way I implemented this I can't give a log of how a single alias is found. (All aliases are flattened first before being looked up).

Reply:
At least knowing the visited catalogs would be helpful. Having it per alias can be future thing if ever asked :)